packages/dom/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 2.1.0 (2019-03-06)
```

Comment:
d1bfd4e 

I ended up using a specific version instead of x.x.x because I want to experiment with `@since` tag, so we can automatically build things like [this](https://developer.wordpress.org/reference/functions/add_theme_support/#changelog).